modules/check_in/app/sidekiq/check_in/travel_claim_notification_job.rb

```diff
@@ -13,25 +17,25 @@ module CheckIn
   #     '1234'                                # Required - Last four digits of claim number
   #   )
   class TravelClaimNotificationJob < TravelClaimBaseJob
-    sidekiq_options retry: 12
+    sidekiq_options retry: 1
```

Comment:
Note: this change snuck in from my local testing, change before submitting for review. If you are reviewing and see this my apologies and please lmk